tests/test_api_v1.py

```diff
@@ -512,6 +541,20 @@ def test_create_request_connection_error(mock_chain, app, auth_env, client, db):
     assert rv.json == {"error": "Failed to connect to the broker to schedule a task"}
 
 
+def test_create_request_unsupported_pkg_manager(app, auth_env, client, db):
```

Comment:
"unsupported" is misleading. It's support, just not enabled. Will there be a different error message if an unsupported package manager is provided?

Reply:
Good point. A different error does occur for a package manager that is not supported. This occurs in `cachito.web.models.py::PackageManager.get_pkg_managers`.

I'll rename this test.